Mailbox/Person.cs

```diff
@@ -1,4 +1,47 @@
-﻿namespace Mailbox
+﻿/*
+#### Person
+- Create a new `Person` struct
+- A person has a first and last name
+- `Person` should implement `IEquatable<Person>`
+- Ensure appropriate equality operators are also written
+- Implement a reasonable ToString method
+    */
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    public struct Person : IEquatable<Person>
+    {
+        public string FirstName { get; }
+        public string LastName { get; }
+
+        public Person(string firstName, string lastName)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+        }
+
+        public override int GetHashCode() => (FirstName, LastName).GetHashCode();
+
+        public override bool Equals(object? obj)
+        {
+            if (obj is Person)
```

Comment:
A better approach would be pattern matching.

```suggestion
            if (obj is Person person ) return Equals(person);
```